lib/jdomodels/src/main/resources/schema/MembershipRequest-ddl.sql

```diff
@@ -1,12 +1,12 @@
-CREATE TABLE IF NOT EXISTS `MEMBERSHIP_REQUEST_SUBMISSION` (
-  `ID` bigint(20) NOT NULL,
-  `TEAM_ID` bigint(20) NOT NULL,
-  `CREATED_ON` bigint(20) NOT NULL,
-  `USER_ID` bigint(20) NOT NULL,
-  `EXPIRES_ON` bigint(20),
-  `PROPERTIES` mediumblob,
-  PRIMARY KEY (`ID`),
-  CONSTRAINT `MEMBERSHIP_REQUEST_TEAM_FK` FOREIGN KEY (`TEAM_ID`) REFERENCES `TEAM` (`ID`) ON DELETE CASCADE,
-  CONSTRAINT `MEMBERSHIP_REQUEST_USER_FK` FOREIGN KEY (`USER_ID`) REFERENCES `JDOUSERGROUP` (`ID`)
-  
+CREATE TABLE IF NOT EXISTS `MEMBERSHIP_REQUEST_SUBMISSION` (
```

Comment:
Once we've verified that migration works, change the table name to MEMBERSHIP_REQUEST, please.

Reply:
Tracking this task in https://sagebionetworks.jira.com/browse/PLFM-4716